README.md

```diff
@@ -3,7 +3,7 @@ cloudify-aws-plugin
 
 A Cloudify Plugin that provisions resources in Amazon Web Services
 
-* Master Branch [![Build Status](https://travis-ci.org/cloudify-cosmo/cloudify-aws-plugin.svg?branch=master)](https://travis-ci.org/cloudify-cosmo/cloudify-aws-plugin)
+* CFY-1892 Branch [![Build Status](https://travis-ci.org/cloudify-cosmo/cloudify-aws-plugin.svg?branch=CFY-1892)](https://travis-ci.org/cloudify-cosmo/cloudify-aws-plugin)
```

Comment:
it deliberately states "master" branch. We always show it and only it. it's fine.. you don't have to change this for every branch.